bin/cli.ml

```diff
@@ -41,9 +41,9 @@ let section =
   named (fun x -> `Section x)
     Arg.(value & opt (some string) None & info ["section"; "s"] ~doc ~docv:"PAT")
 
-let not_verbose =
+let verbose =
```

Comment:
I think it should still be called `not_verbose`, to be consistent with the document and the value it generates